### PR TITLE
Moved ascension to a method

### PR DESCRIPTION
**Pull request by Hornsoup**
Can now be used for empty ascension if a player just wants to ascend and reset aptitudes, and for other possible ascensions.

### CODE REVIEW

File: src/app/game-state/character.service.ts [hunk 1]
```diff
@@ -147,20 +147,7 @@ export class CharacterService {
...
-    }
-    this.logService.addLogMessage(
-      "Your spirituality coelesces around the core of your soul, strengthening it and reforging it into something stronger.",
-      'STANDARD', 'STORY');
-    this.logService.addLogMessage(
-      "You now gain additional aptitude each time you reincarnate.",
-      'STANDARD', 'STORY');
-    this.characterState.condenseSoulCoreCost *= 10;
-    this.characterState.aptitudeGainDivider -= 10;
+  ascend(){
```
Comment (immortalityidle) [inline src/app/game-state/character.service.ts:150]: Nitpick on the name of the function here. Maybe resetAttributes()?

Comment (immortalityidle) [reply src/app/game-state/character.service.ts:150]: Or resetAptitudes()?

Comment (Hornsoup) [reply src/app/game-state/character.service.ts]: Added. It's a better naming standard.

File: src/app/game-state/character.service.ts [hunk 2]
```diff
@@ -205,24 +208,7 @@ export class CharacterService {
...
-      } else {
-        attribute.value = 0;
-      }
-    }
-    if (!this.activityService){
-      this.activityService = this.injector.get(ActivityService);
-    }
-    this.activityService.reloadActivities();
-    this.activityService.activityLoop.splice(0, this.activityService.activityLoop.length);
-    this.forceRebirth = true;
```
[the remark this answers is not in the document]
Comment (immortalityidle) [reply src/app/game-state/character.service.ts:224]: I think if they ascend in the middle of an impossible task they might get messed up if they don't have this bit.

[the remark this answers is not in the document]
Comment (ChandlerFerry) [reply src/app/game-state/character.service.ts:224]: Thanks, sheds some light on what it does.

**Review by ChandlerFerry — APPROVED** [review]

**Review by immortalityidle — APPROVED** [review]

**Pull request merged**